app/controllers/support/devices/order_status_controller.rb

```diff
@@ -21,6 +27,12 @@ def update
     render :edit, status: :unprocessable_entity
   end
 
+  # GET /support/devices/schools/:urn/enable-orders/confirm
```

Comment:
What's the easiest way to prevent this from falling over locally because the CC API is not set up? Is it worth having a guard on the CC call so it's only enabled in the environments where it's configured?

Reply:
Would a FeatureFlag be easiest like the email notifications?

Reply:
A FeatureFlag makes most sense to me - I've added it in fcb5451f